frontend/android/buildSrc/src/main/kotlin/BuildConfig.kt

```diff
@@ -43,7 +43,7 @@ private val json = Json { ignoreUnknownKeys = true }
 
 fun readBuildConfig(buildConfigName: String): BuildConfig {
     val jsonString = CommandLine.execute(
-        listOf("app-toolbelt", "v0", "build-config", "to-json", buildConfigName, "android"),
+        listOf("../../node_modules/app-toolbelt/bin/app-toolbelt", "v0", "build-config", "to-json", buildConfigName, "android"),
```

Comment:
I think it would be better to use `npx --no app-toolbelt ...`  instead of manually selecting the path.

Reply:
I was about to ask why it was necessary :D That makes a lot of sense, thanks :)

Reply:
That would work too. I think you can even do
```json
  "app-toolbelt": "app-toolbelt"
```

Reply:
But you would have to pass the arguments after a double dash:
```
npm run app-toolbelt -- v0 [...]
```

Reply:
Now I run into problems because the working directory is now the project root. Any tips to circumvent that? :D

Reply:
I guess, that's better, then. 👍🏽